src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -161,9 +161,6 @@ static int execute_host_command(
     if (code != StatusCode::Success)
         return code;
 
-    // Previous hostfxr trace messages must be printed before calling trace::setup in hostpolicy
-    trace::flush();
-
```

Comment:
I don't understand why we need to remove the flush here?
All the other cases have it for a good reason - this one as well...

Reply:
I think that one accidentally got removed at some point when I was trying to refactor some common code out that I couldn’t do the way I wanted. I’ll revert it